source/2019-12-29-countdown-to-the-new-year-ember-code-snippet.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Countdown to The New Year - Ember Code Snippet
+author: Isaac Lee
+tags: 2019, Addons, DecEmber
+alias: 2019-12-29-countdown-to-the-new-year-ember-code-snippet.md
+responsive: true
+---
+
+**This is the twenty-ninth in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 29
+
+Before I took this assignment, I didn't know anything about [ember-code-snippet](https://emberobserver.com/addons/ember-code-snippet). Now, I'm glad that I did. This addon surfaced in 2015, has received regular updates, and powers many of my favorite addons. (Can you guess how?)
+
+READMORE
+
+### What It Does
+
+[ember-code-snippet](https://github.com/ef4/ember-code-snippet) lets you render code snippets in your app (or addon). Quite handy if you want to create a technical blog or **document your components**.
+
+It provides a helper called `get-code-snippet` so that (1) you can save code snippets in a dedicated folder and (2) create your own component to customize their look.
+
+To illustrate the use, we'll consider a code snippet from [Ember.js Octane vs Classic Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/):
```

Comment:
What do you think about the following?

```suggestion
To illustrate the use, we'll consider a code snippet from [the Ember.js Octane vs Classic Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/):
```

Reply:
Yep, I took the title from their website, but think adding "the" helps the sentence flow better. Thanks!